cmake/toolchain/zephyr/generic.cmake

```diff
@@ -1,3 +1,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
+if(NOT(DEFINED(ENV{SDK_VERSION})))
```

Comment:
Should we be testing the CMake variable, instead of the env var? Since below we are dereferencing a variable?